vuu-ui/packages/vuu-shell/src/theme-provider/ThemeProvider.tsx

```diff
@@ -2,41 +2,55 @@ import React, {
   createContext,
   HTMLAttributes,
   ReactNode,
-  useContext,
   ReactElement,
+  useState,
+  isValidElement,
+  cloneElement,
 } from "react";
 import cx from "classnames";
 
-export const DEFAULT_DENSITY = "medium";
-export const DEFAULT_THEME = "salt-theme";
+export const DEFAULT_DENSITY:Density = "medium";
+export const DEFAULT_THEME:Theme = "salt-theme";
+export const DEFAULT_THEME_MODE:ThemeMode = "light";
 
 export type Density = "high" | "medium" | "low" | "touch";
+export type Theme = "salt-theme";
```

Comment:
I think ultimately theme will need to be a string, as we will want to allow custom / third party themes